### ISSUE TITLE
Crashes when optimizing an old tinypng-optimized image

### ISSUE THREAD

**Issue by datio** [shssoichiro/oxipng#113]
-> [logo.png](https://alex.domakidis.com/old/maths/assets/images/logo.png)

```
➜ oxipng -o 6 -i 1 --strip safe logo.png    
Processing: logo.png
    138x138 pixels, PNG format
    4x8 bits/pixel, RGBA
    IDAT size = 15374 bytes
    File size = 16280 bytes
Reducing image to 2x8 bits/pixel, Grayscale + Alpha
thread '<unnamed>' panicked at 'index 26714 out of range for slice of length 26480', libcore/slice/mod.rs:785:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
thread '<unnamed>' panicked at 'index 26714 out of range for slice of length 26480', libcore/slice/mod.rs:785:5```

**Comment by shssoichiro** [comment]
Thanks! I'll investigate this as soon as possible.

**Comment by shssoichiro** [comment]
This appears to be an issue in alpha optimization for interlaced images. As a temporary workaround, you could use `-o 3` to successfully process the image. I'll get a fix for this out within a few days.

**Comment by shssoichiro** [comment]
Just published in 2.0.2
